README.md

```diff
@@ -257,6 +257,9 @@ Run `M-x customize-group RET doom-modeline RET` or set the variables.
 ;; If non nil the default perspective name is displayed in the mode-line.
 (setq doom-modeline-display-default-persp-name nil)
 
+;; If non nil the perspective name is displayed alongside a folder icon.
+(setq doom-modeline-display-persp-icon t)
```

Comment:
To align other options, I prefer to `doom-modeline-persp-icon`.

Reply:
Updated in b44a483